### DISCUSSION TITLE
getFile() method:  wrong file_id in a local telegram bot api

### DISCUSSION THREAD

**Discussion by sr-44** [nutgram/nutgram#596] [Q&A] [answered]
### Nutgram version

4.9

### PHP version

8.2

### Framework

Laravel

### Cache driver

Laravel

### Describe the bug

I’m using a local bot api and I want to download a file that the user uploaded via 
```php
$audio->thumbnail?->download($path);
``` 
As a result I get an error
`GuzzleHttp\Exception\ClientException: Client error: POST http://example.com/bot123456789:abcd/getFile resulted in a 400 Bad Request response:
{"ok":false,"error_code":400,"description":"Bad Request: wrong file_id or the file is temporarily unavailable"}`

Maybe I don’t quite understand how to work with the local bot api
What to do?

### Expected behavior

.

**Comment by Lukasss93** [comment, marked as answer]
Hi! See this: https://github.com/nutgram/nutgram/discussions/595
